src/test/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorServiceSpec.scala

```diff
@@ -27,23 +27,28 @@ class PolicyEvaluatorServiceSpec extends FlatSpec with Matchers with TestSupport
   lazy val policyDAO: AccessPolicyDAO = new PostgresAccessPolicyDAO(TestSupport.dbRef, TestSupport.blockingEc)
   val schemaDao = new JndiSchemaDAO(directoryConfig, schemaLockConfig)
 
-  private val dummyUserInfo =
+  override protected def beforeEach(): Unit = {
+    setup().unsafeRunSync()
+    super.beforeEach()
+  }
```

Comment:
the tests in this class were not consistent in calling `setup()` which meant many of the tests actually relied on data inserted in a prior test. Using `beforeEach` makes `setup()` consistent and let to other fixes.